file.md

```diff
@@ -19,6 +19,7 @@ Scheme files have the following example structure:
 
     scheme: "Scheme Name"
     author: "Scheme Author"
+    description: "a short description of the scheme"
```

Comment:
it's still missing for config.yaml above

Reply:
This PR is only dealing with schemes, not templates.

Reply:
why then you linked it there? https://github.com/base16-project/base16/issues/3#issuecomment-1172664841

Reply:
Pretty sure you mentioned but I don't see it now. I was thinking schemes the whole time, hence the PR.

Reply:
I don't think it makes much sense to have a description in the config.yaml at the moment - they can be built-in to the templates.